test/inventoryDb/setup.js

```diff
@@ -0,0 +1,36 @@
+// This script tears down all existing test database artifacts for the service,
+// sets them up from scratch and populates the tables with a small test data set.
+
+const { Pool } = require('pg');
+const { database, pgConnection, testDbConnection } = require('./config');
+const { createTable, createIndex } = require('../../inventoryLocalStore/setup/queries');
+const {
+  LISTINGS_TABLE, listingAttributes, listings,
+  AVAILABILITY_TABLE, availabilityAttributes, availability,
+} = require('./fixtures');
+
+// Connect first to the postgres database to manage inventory_test database drop/create
+let pool = new Pool({
+  connectionString: pgConnection,
```

Comment:
const pool.

Reply:
No, pool gets reassigned further down so it can't be a const.